apps/comps/components/providers.tsx

```diff
@@ -57,15 +54,13 @@ function WalletProvider(props: { children: ReactNode }) {
           wallet: siweMessage.address,
```

Comment:
to resolve the linting error, the `const res = await login(...)` should just be `await login(...)` since `res` is no longer used

Reply:
yep, I forgot this one